calicoctl/calicoctl/commands/resourceloader/resourceloader.go

```diff
@@ -200,3 +206,25 @@ func CreateResourcesFromFile(f string) ([]unversioned.Resource, error) {
 
 	return resources, nil
 }
+
+// Kubernetes NetworkPolicy helper struct used for conversion from
+// Kubernetes API to Calico v3 API
+type K8sNetworkPolicy struct {
```

Comment:
This might be a dumb question, but why not just use the K8s definition of the NetworkPolicy here? From what I can tell, it's effectively the same for all we care since it has the same meta and spec.

Reply:
I did try not having this, but I thought it was needed to make it "castable" from `unversioned.Resource`. Am I going about this the wrong way?

Reply:
Nope, you're right in that case. I wasn't sure if the K8s network policy matches the interface that we have for the return values, but if it doesn't, then we'll need to implement it ourselves.

Reply:
Got it, thanks @mgleung! Following our offline chat, I'll clean up the struct of the unnecessary stuff and push